test/documentViewer/pdfViewer/fallback/test.js

```diff
@@ -20,10 +20,10 @@
  */
 define([
     'jquery',
-    'core/promise',
+    
```

Comment:
It's very nice you've removed all references to `core/promise`, it was just deprecated.
However could you please remove the blank line